charts/k8ssandra/templates/traefik.ingressroutes.yaml

```diff
@@ -43,3 +45,92 @@ spec:
         - name: {{ $clusterName }}-{{ $datacenterName }}-service
           port: 9042
 {{- end }}
+{{- if $traefik.stargate.enabled }}
+---
+{{- if and $traefik.stargate.cassandra.enabled $traefik.cassandra.enabled }}
+  {{- fail "ingress.traefik.stargate.cassandra.enabled and ingress.traefik.cassandra.enabled cannot both be enabled." }}
+{{- end }}
+{{- $stargateHost := default "localhost" $traefik.stargate.host }}
```

Comment:
It's in both; this one makes it not blow up if you explicitly set it to an empty string in your value overrides.

Reply:
If I specifically set it to an empty string and the value of `localhost` is rendered it would be confusing. I expect failure in the case of invalid values files.

Reply:
If we are defaulting to `localhost` here, then I think it makes sense to go ahead put that in `values.yaml`.

Reply:
I've changed values.yaml to localhost and replaced the `default` here with `required` so that it blows up if you override with empty.

Reply:
Done.